src/juris-fr-desc.json

```diff
@@ -1,64 +1,99 @@
 {
   "courts": {
-    "ca": {
-		"name": "Cour d'appel",
-		"abbrev": "CA >"
-    },
     "cass": {
-      "name": "Cour de Cassation"
+      "name": "Cour de cassation",
+	"abbrev": "Cass."	    
     },
-    "cass~civ1": {
-      "name": "Cour de Cassation|1re chambre civile"
+    "cass~civ.1": {
+      "name": "< 1re Chambre civile"
+	    "abbrev": "< civ. 1re"
     },
-    "cass~civ2": {
-      "name": "Cour de Cassation|2e chambre civile"
+    "cass~civ.2": {
```

Comment:
The period in `civ.2` mit lead to errors. It’s just a code so no need to make it look nicer :)

Reply:
I noticed the ECJ followed this format, I wasn't sure whether or not it was standard!

Reply:
Right, I overlooked that.